packages/flutter_tools/test/general.shard/devfs_test.dart

```diff
@@ -205,79 +195,9 @@ void main() {
     });
 
     tearDownAll(() async {
-      await vmService.tearDown();
       _cleanupTempDirs();
     });
 
-    testUsingContext('create dev file system', () async {
```

Comment:
We do not use the vm_service based-devfs. Base64 encoding files to be sent through JSON rpc is too slow